vehicle-tracker/vehicle_tracker_app/.env

```diff
@@ -1,5 +1,6 @@
 API_URL=https://uat.digit.org/egov-location/
 LOCALIZATION_API_URL=https://health-qa.digit.org/localization/messages/v1/_search?
+MDMS_URL=https://raw.githubusercontent.com/egovernments/egov-mdms-data/UNIFIED-DEV/data/pb/common-masters/StateInfo.json
```

Comment:
This should not be a json path :
Fix : {baseURL}/egov-mdms-service/v1/_search?tenantId=pb
and pass the modules to get the data

Reply:
Will require the exact url, I am getting this response

```
{
  "ResponseInfo": null,
  "Errors": [
    {
      "code": "NotNull.mdmsCriteriaReq.mdmsCriteria",
      "message": "must not be null",
      "description": null,
      "params": null
    },
    {
      "code": "NotNull.mdmsCriteriaReq.requestInfo",
      "message": "must not be null",
      "description": null,
      "params": null
    }
  ]
}
```